cmake/kconfig.cmake

```diff
@@ -104,6 +104,38 @@ if(EXTRA_KCONFIG_OPTIONS)
     )
 endif()
 
+# Set overlay config files list using a list of directories where the
+# overlay files could be found.
+if(DEFINED ENV{OVERLAY_CONFIG_DIRS})
```

Comment:
If we support env vars, we should also support setting this through a CMake var.